docs/api-testing/integrations/pagerduty-webhooks.md

```diff
@@ -69,11 +69,11 @@ To test a connector:
 2. Click on any project.
 3. Within that project, click **Settings** and then click **Connector**.
 4. On the **Connectors** page, click **Test Connectors**.
-<img src={useBaseUrl('img/api-testing/test-connectors-nav.png')} alt="Test Connectors button"/>
+<img src={useBaseUrl('img/api-testing/test-connectors-nav.png')} alt="Test Connectors button" width="600"/>
 5. In the **Connectors Test Drive** window, select either the **Failure** or **Success** radio button, and then click **Trigger**.
-  * Failure - You will receive a notification in all connector platforms in your list
-  * Success - If the **On_success** toggle is set to **True**, you will receive a notification for the connector
-<img src={useBaseUrl('img/api-testing/trigger-confirmation.png')} alt="Test Connectors dialog box"/>
+   * Failure - You will receive a notification in all connector platforms in your list
+   * Success - If the **On_success** toggle is set to **True**, you will receive a notification for the connector<br/>
+<img src={useBaseUrl('img/api-testing/trigger-confirmation.png')} alt="Test Connectors dialog box" width="400"/>
```

Comment:
I'd make these a bit bigger, like 700 / 500. I tested it and they're a bit small.